webhook_to_fedora_messaging/models/user.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import Column, Integer, UnicodeText, Boolean
+from webhook_to_fedora_messaging.models.util import UUIDCreatableMixin, CreatableMixin
+
+from webhook_to_fedora_messaging.database import Base
+
+
+class User(Base, UUIDCreatableMixin, CreatableMixin):
+    __tablename__ = "users"
+
+    id = Column("id", Integer, primary_key=True, nullable=False)
+    username = Column("username", UnicodeText, unique=True, nullable=False)
+    is_admin = Column("is_column", Boolean, nullable=False, default=False)
```

Comment:
I think an enabled/disabled switch would be useful :-)

Reply:
You missed out the `is_column` oopsie that I named the column as which could have been meme'd to hell but something tells me that you did it on purpose. You have my gratitude.